packages/react-components/react-shared-contexts/src/CustomStyleHooksContext/CustomStyleHooksContext.ts

```diff
@@ -90,8 +90,8 @@ export type CustomStyleHooksContextValue = Partial<{
   useDataGridHeaderCellStyles_unstable: CustomStyleHook;
   useDataGridSelectionCellStyles_unstable: CustomStyleHook;
   useDrawerStyles_unstable: CustomStyleHook;
-  useDrawerInlineStyles_unstable: CustomStyleHook;
-  useDrawerOverlayStyles_unstable: CustomStyleHook;
```

Comment:
discussed offline with @marcosmoura 

we need to keep both old and new hooks until v10.

**Why:**

- this package is already stable, removing things from public api === breaking change
- this will need to live here even after card is released as stable
- card needs to maintain proper order of calling these hooks ( `useDrawerOverlayStyles_unstable` then ` useOverlayDrawerStyles_unstable` )

In general this is not the best architectural decision as we create **indirect tight coupling** that cannot be checked if it works ( we need to provide integration tests to this package ).

This issues can happen also with our new release strategy concept for unstable packages (`*-preview`)

**Action points:**

- create RFC describing the issue
- present at tech sync